Docs/articles/Kernel/Startup.md

```diff
@@ -1,3 +1,5 @@
 On startup, there is some hand-coded assembly that runs before the Cosmos layer kicks in. From there, the C# entry point Cosmos.System.Kernel.Start() is called.
 
 Cosmos.System.Kernel is an abstract class that forms the Cosmos framework upon which your OS is built upon.
+
+You Can override Kernel.Start() method and add your own code.
```

Comment:
Rephrase, "You can override the Kernel.Start() method to suppress the standard Cosmos boot routines."

Reply:
Thank you for helping me in my phrase, I Rephrased as reqested